packages/nimble-components/src/table-column/icon/cell-view/template.ts

```diff
@@ -0,0 +1,19 @@
+import { html, when } from '@microsoft/fast-element';
+import type { TableColumnIconCellView } from '.';
+import { spinnerTag } from '../../../spinner';
+
+export const template = html<TableColumnIconCellView>`
+    ${when(
```

Comment:
Could you do:
```
   ${when(x => x.iconTemplate, html`
        {x => x.iconTemplate}
    `)}
    ${when(x => !x.iconTemplate,
        html<TableColumnIconCellView>`
        <${spinnerTag}
            title="${x => x.text}"
            aria-label="${x => x.text}">
        </${spinnerTag}>
    `
    )}
```

Basically just trying to get rid of the redundant `visual` observable (this would also require making `iconTemplate` an `observable`).

Reply:
Realized I can't do this. There is a third state where the value doesn't match any mappings, and we don't show either template. Switched back to using the `visual` property.